addons/map_gestures/ACE_Settings.hpp

```diff
@@ -0,0 +1,50 @@
+class ACE_Settings {
+    class GVAR(enabled) {
+        value = 1;
+        typeName = "BOOL";
+        displayName = CSTRING(enabled_displayName);
+        description = CSTRING(enabled_description);
+    };
+    class GVAR(maxRange) {
+        value = 7;
+        typeName = "SCALAR";
+        displayName = CSTRING(maxRange_displayName);
+        description = CSTRING(maxRange_description);
+    };
+    class GVAR(interval) {
+        value = 0.03;
+        typeName = "SCALAR";
+        displayName = CSTRING(interval_displayName);
+        description = CSTRING(interval_description);
+    };
+    class GVAR(defaultLeadColor) {
```

Comment:
I agree, colours should be set by the client for users who have colour-specific needs.